bytes_test.go

```diff
@@ -130,6 +132,9 @@ func TestBytes(t *testing.T) {
 		{"bytes(5.5GiB)", IBytes(5.5 * GiByte), "5.5 GiB"},
 
 		{"bytes(5.5GB)", Bytes(5.5 * GByte), "5.5 GB"},
+
+		{"bytes(123456789, 3)", IBytesN(123456789, 3), "118 MiB"},
+		{"bytes(123456789, 6)", IBytesN(123456789, 6), "117.738 MiB"},
```

Comment:
This example is helpful.  I didn't quite understand how this was meant to work before.  It seems useful for printing meaningful numbers in a way I've not seen considered.

Is it possible to make that clearer in the docs?  I found the idea bit confusing without the example, so maybe the example is helpful?

Reply:
Sure! I'll add some examples to the Godoc to make the feature more obvious.

Reply:
Added the examples and reworded the documentation now.